src/Button/CopyButton/CopyButton.tsx

```diff
@@ -56,7 +56,7 @@ const CopyButton: React.FC<CopyButtonProps> = ({
     if (digitizeLayer) {
       setLayers([digitizeLayer]);
     } else {
-      setLayers([DigitizeUtil.getDigitizeLayer(map)]);
+      setLayers([DigitizeUtil.getDigitizeLayer(map) as any]);
```

Comment:
why is this needed? Are the types of map not compatible? If so we should look if we can do something about it.

Reply:
Fixed in latest amend